pkgs/development/tools/build-managers/cmake/cxxtest_cmake.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Modules/FindCxxTest.cmake	2016-01-25 17:57:19.000000000 +0100
++++ b/Modules/FindCxxTest.cmake	2016-08-01 16:26:30.940172420 +0200
+@@ -221,7 +221,7 @@
+ 
+    if(PYTHONINTERP_FOUND AND (CXXTEST_USE_PYTHON OR NOT PERL_FOUND OR NOT DEFINED CXXTEST_USE_PYTHON))
+       set(CXXTEST_TESTGEN_EXECUTABLE ${CXXTEST_PYTHON_TESTGEN_EXECUTABLE})
+-      set(CXXTEST_TESTGEN_INTERPRETER ${PYTHON_EXECUTABLE})
++      set(CXXTEST_TESTGEN_INTERPRETER "/bin/sh")
```

Comment:
@juliendehos Please use the value of the Nix variable `stdenv.shell` instead of hardcoding to `/bin/sh`. `/bin/sh` may not be the same shell that the wrapper expects!